<launch>

  <!-- send pr2_l_arm.xml to param server -->
  <param name="robot_description" command="$(find xacro)/xacro.py '$(find meka_description)/robots/uja_fixed_hands.urdf.xacro'" />

  <node pkg="rviz" type="rviz" respawn="false" output="screen" name="rviz"
   args="-d '$(find meka_description)/m1.rviz'" required="true" />

  <!-- Robot state publisher -->
  <node pkg="robot_state_publisher" type="state_publisher" name="robot_state_publisher">
    <remap from="joint_states" to="humanoid_state" />
  </node>

</launch>

